zvmsdk_rhel.spec

```diff
@@ -35,6 +35,7 @@ mkdir -p %{buildroot}/var/opt/zvmsdk
 cp zvmsdklogs %{buildroot}/var/opt/zvmsdk
 cp tools/share/zvmguestconfigure  %{buildroot}/var/lib/zvmsdk/
 cp tools/share/zvmguestconfigure.service %{buildroot}/var/lib/zvmsdk/
+cp tools/share/zvmguestconfigure.service.ubuntu %{buildroot}/var/lib/zvmsdk/
```

Comment:
check line 46.
pls double check all zvmguestconfigure.service operations are also done for zvmguestconfigure.service.ubuntu.

Reply:
thanks, add in line 46. Search in all directory, not find other places.